electrum/keystore.py

```diff
@@ -58,6 +58,8 @@
 
 class CannotDerivePubkey(Exception): pass
 
+class ScriptTypeNotSupported(Exception): pass
```

Comment:
This is already defined in base_wizard.py.
Speaking of; can base_wizard.py be deleted?
How about all the overrides of the old wizard methods in trustedcoin.py?

Reply:
I avoided referencing `base_wizard.py` as that file is planned to become unused as soon as the kivy client is removed. `trustedcoin.py` still has some leftovers which I will remove.

Reply:
trustedcoin old wizard leftovers removed.